binding/Binding/SKTypeface.cs

```diff
@@ -108,6 +110,46 @@ public int CharsToGlyphs (IntPtr str, int strlen, SKEncoding encoding, out ushor
 				}
 			}
 		}
-	}
+
+        public string FamilyName {
+            get {
+                var r = SkiaApi.sk_string_new_empty();
+                try {
+                    SkiaApi.sk_typeface_get_family_name(Handle, r);
+                    var cstr = SkiaApi.sk_string_get_c_str(r);
+                    var clen = SkiaApi.sk_string_get_size(r);
+                    return Util.GetString(cstr, (int)clen, SKTextEncoding.Utf8); 
+                } finally {
+                    SkiaApi.sk_string_destructor(r);
```

Comment:
If we return a SKString from unmanaged code, we won't have to do this each time. Just get the IntPtr, and then get the SKString... the GC will do the rest